### PR TITLE
Fix(app):  fix ctx debug [#7928]

### PR DESCRIPTION
**Pull request by martijn788**
**What kind of change does this PR introduce?** (check at least one)

- [X] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [X] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [X] It's submitted to the `dev` branch and _not_ the `master` branch
- [X] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

related to: 
https://github.com/quasarframework/quasar/commit/ec2cd27a2c19dd6026cbf0ec2aa0de2972619ab3
https://github.com/quasarframework/quasar/issues/7911

### CODE REVIEW

File: app/templates/entry/client-entry.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ import electron from 'electron'
 Vue.prototype.$q.electron = electron
 <% } %>
 
-<% if (ctx.debug) { %>
+<% if (ctx.dev || ctx.debug) { %>
```
Comment (hawkeye64) [inline app/templates/entry/client-entry.js:69]: This is not needed, since `ctx.debug` implies dev build or production build with --debug parameter

Comment (martijn788) [reply app/templates/entry/client-entry.js:69]: There productionTip is now showing up in the console when you run quasar dev

Comment (hawkeye64) [reply app/templates/entry/client-entry.js:69]: yes, it should be `false`, the way it was when it was just `if (ctx.dev)`

Comment (martijn788) [reply app/templates/entry/client-entry.js:69]: What do you mean? 

I tested two scenarios:
if (ctx.debug) --> productiontip is shown
if (ctx.debug || cyx.dev) --> productiontip is hidden


process.env.DEBUGGING is set with the same setter. (ctx.debug || ctx.dev)
That's why i thought that it should be the same here. 

Do you know where the issue from the productiontip is coming from?

Comment (hawkeye64) [reply app/templates/entry/client-entry.js:69]: You are right. Sorry about my own confusion. :)
Reopening...

Comment (martijn788) [reply app/templates/entry/client-entry.js:69]: No problem!
I was thinking the same before I tested it:)

Do I also need to make an issue for this?

Comment (hawkeye64) [reply app/templates/entry/client-entry.js:69]: no
https://discord.com/channels/415874313728688138/599146158828421140/768841045420408853

**Comment by rstoenescu** [comment]
Thanks for taking time for this PR!

**Pull request merged**